pandas/tests/indexes/test_multi.py

```diff
@@ -1425,8 +1448,6 @@ def test_equals_multi(self):
 
         self.assertFalse(self.index.equals(self.index[:-1]))
 
-        self.assertTrue(self.index.equals(self.index._tuple_index))
```

Comment:
Yes, I understand that, but we still want to test the fact that Index of tuples evaluates as equal.